python/ray/tests/test_runtime_uv.py

```diff
@@ -0,0 +1,64 @@
+# TODO(hjiang): A few unit tests to add after full functionality implemented.
+# 1. Install specialized version of `uv`.
+# 2. Options for `uv install`.
+# 3. Use requirement files for packages.
+
```

Comment:
Most pip integration test can have a uv counterpart. Consider reusing the tests by using `@pytest.mark.parametrize`. This also shows people can just change `pip` to `uv` and things will just work.

Reply:
Yeah I considered that, but choose to rewrite in these reasons:
- Not all the features are implemented for `uv` for now, so quite a lot of existing pip-based tests will be skipped;
- pip and uv installation process is different: we consider pip installed by default, while have to install uv ourselves, which means, we need to do extra checking for `uv`
- uv and pip are not 100% compatible, need to add uv-specific tests at the end anyway